gm4_metallurgy/data/gm4_metallurgy/predicates/infinitas_active.json

```diff
@@ -5,14 +5,20 @@
       "condition": "minecraft:entity_properties",
       "entity": "this",
       "predicate": {
-        "nbt": "{Tags:[\"gm4_infinitas_mainhand_empty\"]}"
+        "nbt": "{Tags:[\"gm4_infinitas_mainhand_empty\"]}",
+        "flags": {
+          "is_sneaking": true
+        }
```

Comment:
I don't fully understand this part of the module, but why was this change necessary?

Reply:
_This part was actually unrelated and a server test.  It probably should have been put in a different PR :grimacing:_

Reply:
The reason it was added: Previously, Spiraculum was not usable as a tool, so you wouldn't be holding a Spiraculum bucket unless you were specifically looking for the vacuum effect.  Now, with empty Infinitas Buckets having a purpose, it becomes very frustrating when you frequently vacuum nearby entities.  Sneaking to trigger the vacuum effect seemed like a solid way to activate it (and it felt pretty good to use when I tested it on the server)